### ISSUE TITLE
Start timer return incorrect values. Angular 12 with version 2.2.6

### ISSUE THREAD

**Issue by toski112** [rednez/angular-user-idle#115]
Hi!

I have a problem with the new 2.2.6 version. I use angular 12. With version 2.2.4 all is working fine, but in the newest version 2.2.6 onTimerStart stream is returning incorrect values (not starting from 1 like older version).

I prepared screenshots and stackblitz. Can you tell me how to fix it? In my actual implementation I waiting for value 1 but in version 2.2.6 is not starting from 1.....
Thanks!

Stackblitz:
https://stackblitz.com/edit/angular-12-form-validation-qeprdd

onTimerStart in version 2.2.4:
![2 2 4](https://user-images.githubusercontent.com/29277360/128168902-365b87f1-969a-4a2f-82c5-cba58f591072.PNG)

onTimerStart in version 2.2.6:
![2 2 6](https://user-images.githubusercontent.com/29277360/128168914-6b491aac-dea3-44ac-87d6-2ddc6be1acd9.PNG)

In my opinion error is caused by this change from 2.2.6 (stream is counting with 3 seconds delay):

FROM ->
```
      this.timer$ = interval(1000).pipe(
        take(timeout),
        map(() => 1),
        scan((acc, n) => acc + n),
        tap(count => {
          if (count === timeout) {
            this.timeout$.next(true);
          }
        })
      );
```

TO -> 

```

      this.timer$ = of(new Date()).pipe(
        switchMap(startDate => interval(1000).pipe(
          map(() => Math.round((new Date().valueOf() - startDate.valueOf()) / 1000)),  //   convert elapsed count to seconds
          tap(elapsed => {
            if (elapsed >= timeout) {
              this.timeout$.next(true);
            }
          }),
          )
        ));
```

**Comment by superjulius** [comment]
@toski112 The problem has been solved in v2.2.7

**Comment by blaiseAI** [comment]
> @toski112 The problem has been solved in v2.2.7. I faced the same issue and confirmed that it works well with the latest version.
> 
> @rednez You can close this issue.

Thanks @toski112 I had exactly the same problem and after updating to v2.2.7, the issue got resolved right away, Thanks guys!!!
